hadoop-tools/hadoop-azure/pom.xml

```diff
@@ -194,7 +203,8 @@
     <dependency>
       <groupId>org.wildfly.openssl</groupId>
       <artifactId>wildfly-openssl</artifactId>
-      <scope>runtime</scope>
+      <version>1.0.9.Final</version>
```

Comment:
Wildfly jar version is picked from the parent project. Retain the original config.

Reply:
I was seeing compilation failure with 1.0.7.Final